### PR TITLE
Jarry/small bug in create mint

### PR DESCRIPTION
**Pull request by jarry-xiao**
unwrap previously leading to CLI panic

### CODE REVIEW

File: src/lib/helpers/print_helpers.rs [hunk 1]
```diff
@@ -182,26 +182,34 @@ pub async fn print_market_details(
     println!("Quote vault key: {}", market_header.quote_params.vault_key);
 
     println!(
-        "Base Lot Size, in raw base units: {}",
+        "Raw base units per base lot: {}",
         get_decimal_string(
             market_metadata.base_atoms_per_base_lot,
             market_metadata.base_decimals
         ),
```
Comment (throwbackjams) [inline src/lib/helpers/print_helpers.rs:189]: The text says `raw base units per base lot` but the number to be printed is the `base_atoms_per_base_lot`?

File: src/lib/helpers/print_helpers.rs [hunk 2]
```diff
@@ -182,26 +182,34 @@ pub async fn print_market_details(
...
     );
     println!(
-        "Quote Lot Size, in quote units: {}",
+        "Quote units per quote lot: {}",
         get_decimal_string(
             market_metadata.quote_atoms_per_quote_lot,
             market_metadata.quote_decimals
         )
```
Comment (throwbackjams) [inline src/lib/helpers/print_helpers.rs:196]: Same here - mismatch in the text and the number?

File: src/lib/helpers/print_helpers.rs [hunk 3]
```diff
@@ -182,26 +182,34 @@ pub async fn print_market_details(
...
     );
     println!(
-        "Tick size in quote atoms per base unit: {}",
+        "Tick size in quote units per base unit: {}",
         get_decimal_string(
             market_metadata.tick_size_in_quote_atoms_per_base_unit,
             market_metadata.quote_decimals
         )
     );
```
Comment (throwbackjams) [inline src/lib/helpers/print_helpers.rs:204]: Same thing - mismatch in the next and the value?

**Review by throwbackjams — APPROVED** [review]

File: src/lib/helpers/print_helpers.rs (hunk 1 above)
Comment (jarry-xiao) [reply src/lib/helpers/print_helpers.rs:189]: The actually printed value is a decimal string though

**Pull request merged**